src/test/compatibility/custom-json-formatter.js

```diff
@@ -1,14 +1,30 @@
 const { JsonFormatter } = require('cucumber')
+const { notIsNilOrEmpty } = require('../../web/routes/application/common/predicates')
 
 require('dotenv').config()
 
+function isMobileDevice () {
+  return !!process.env.BROWSER_STACK_DEVICE
+}
+
+function getBrowserVersion () {
+  const browserVersion = process.env.BROWSER_STACK_BROWSER_VERSION
+  if (notIsNilOrEmpty(browserVersion)) {
```

Comment:
I probably wouldn't use `notIsNilOrEmpty` here. Javascript string coercion means this is the same as:
```
if (browserVersion) { ... }
```
which I think is a lot less confusing. `notIsNilOrEmpty` is useful when composing functions but I think a simpler approach is better here